snap/snapcraft.yaml

```diff
@@ -0,0 +1,82 @@
+name: xmrig
+adopt-info: xmrig
+summary: XMRig is a high performance, open source, cross platform CPU/GPU miner
+description: |
+  XMRig is a high performance, open source, cross platform RandomX, KawPow, 
+  CryptoNight and AstroBWT unified CPU/GPU miner and RandomX benchmark. 
+  
+  To run, open a terminal:
+    - Ensure you've identified the options you'd like to run
+    - Identify the pool (and port) desired
+    - Use your wallet address
+      - Example: `xmrig -o example.pool.com:4444 -u [wallet address]`
+    - Hit enter and profit
+    
+license: GPL-3.0
+
+base: core20
+grade: stable
+confinement: strict
+compression: lzo
+
+architectures:
+  - build-on: amd64
+    run-on: [amd64, armhf, arm64, ppc64el, s390x]
+
+assumes:
+  - command-chain
+
+apps:
+  xmrig:
+    command: bin/xmrig
+    command-chain: 
+      - bin/homeishome-launch     
+    plugs:
+      - home
+      - network
+      - network-bind
+      - opengl
+      
+parts:
+  xmrig:
+    source: https://github.com/xmrig/xmrig
+    source-type: git
+    plugin: nil
+    
+    override-build: |
+      git clone https://github.com/xmrig/xmrig.git
+      cd xmrig/scripts
+      ./build_deps.sh
+      cmake .. -DXMRIG_DEPS=scripts/deps
+      make -j$(nproc) 
+      mkdir -p $SNAPCRAFT_PART_INSTALL/bin/  
+      cp xmrig $SNAPCRAFT_PART_INSTALL/bin/
+      
+    override-pull: |
+      snapcraftctl pull
+      snapcraftctl set-version "$(git describe --tags | sed 's/^v//' | cut -d "-" -f1)"
+      
+    build-packages:
+      - build-essential 
+      - git
+      - cmake
+      - automake
+      - libtool
+      - autoconf
+      - libuv1-dev 
```

Comment:
Remove all `-dev` packages, not needed (or wanted) when using bundled.

Reply:
These are build packages, not actually bundled with the snap. Are you saying that the following are not needed during runtime?
```
      - libuv1-dev 
      - libssl-dev 
      - libhwloc-dev
```

Reply:
Correct, bundled libs are equivalent to those three packages.